stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/SectionUI.kt

```diff
@@ -31,21 +31,17 @@ fun Section(
     error: String?,
     modifier: Modifier = Modifier,
     isSelected: Boolean = false,
-    addVerticalPadding: Boolean = true,
     contentOutsideCard: @Composable () -> Unit = {},
     contentInCard: @Composable () -> Unit
 ) {
-    val verticalPadding = remember(addVerticalPadding) {
-        if (addVerticalPadding) 8.dp else 0.dp
-    }
-
-    Column(modifier = modifier.padding(top = verticalPadding)) {
-        SectionTitle(title)
-        SectionCard(
-            isSelected = isSelected,
-            content = contentInCard,
-            modifier = Modifier.padding(bottom = verticalPadding)
-        )
+    Column {
+        Column(modifier = modifier) {
+            SectionTitle(title)
+            SectionCard(
+                isSelected = isSelected,
+                content = contentInCard,
+            )
+        }
```

Comment:
This is a little weird to me. We've previously been applying the padding around the title and card but not its accompanying error (if one exists) and any content outside the card.

Reply:
I think I'm the one that added this padding and I'm not sure why I did it this way